appcues/src/main/java/com/appcues/trait/BackdropDecoratingTrait.kt

```diff
@@ -5,6 +5,14 @@ import androidx.compose.runtime.Composable
 
 interface BackdropDecoratingTrait : ExperienceTrait {
 
+    companion object {
+        const val BACKDROP_KEYHOLE_PRIORITY = 99
+        const val BACKDROP_BACKGROUND_PRIORITY = 89
+        const val BACKDROP_SKIPPABLE_PRIORITY = 59
+    }
+
+    val priority: Int
```

Comment:
this works, but still wondering if we can do anything to hide this from the public API at all, since Traits are a public extensibility point. For the priority levels, can they be more semantic? like "HIGH", "DEFAULT", "LOW"?  Alternatively, if we just have the one special case - the idea was to just recognize the `BackdropKeyholeTrait` internally as special case and not expose this priority publicly until we found we needed to? I can see how a custom trait might need it as well though and perhaps some public `BackdropClippingTrait` interface could be used to allow those to execute before other backdrop decorations?

Reply:
I could simplify and have this special case for the keyhole for now until we see another scenario we might need priority or another trait there.